template/src/ontology/run.sh.jinja2

```diff
@@ -74,7 +74,7 @@ if [ x$ODK_DEBUG = xyes ]; then
     # If you wish to change the format string, take care of using
     # non-breaking spaces (U+00A0) instead of normal spaces, to
     # prevent the shell from tokenizing the format string.
-    echo "Running ${IMAGE} with ${ODK_JAVA_OPTS} of memory for ROBOT and Java-based pipeline steps."
+    echo "Running obolibrary/${ODK_IMAGE}:${ODK_TAG} with ${ODK_JAVA_OPTS} of memory for ROBOT and Java-based pipeline steps."
```

Comment:
While you’re at it, you should also change the phrasing, because `${ODK_JAVA_OPTS} of memory` results in something like `-Xmx12G of memory`, which does not make much sense. And people could always use `ODK_JAVA_OPTS` to pass other Java options unrelated to memory.

I’d suggest simply `with ${ODK_JAVA_OPTS} options for ROBOT and…`.